client/a8c-for-agencies/sections/marketplace/lib/product-filter.ts

```diff
@@ -0,0 +1,19 @@
+import {
+	PRODUCT_FILTER_KEY_CATEGORIES,
+	PRODUCT_FILTER_KEY_PRICES,
+	PRODUCT_FILTER_KEY_TYPES,
+} from '../constants';
+
+export type SelectedFilters = {
+	[ PRODUCT_FILTER_KEY_CATEGORIES ]: Record< string, boolean >;
+	[ PRODUCT_FILTER_KEY_TYPES ]: Record< string, boolean >;
+	[ PRODUCT_FILTER_KEY_PRICES ]: Record< string, boolean >;
+};
+
+export function hasSelectedFilter( selectedFilters: SelectedFilters ) {
+	return Object.keys( selectedFilters ).some( ( key ) => {
+		return Object.values( selectedFilters[ key as keyof SelectedFilters ] ).some(
+			( selected ) => selected
+		);
+	} );
```

Comment:
Just a nitpick, can we write it as this:

```suggestion
	return Object.values( selectedFilters ).some( ( filter ) => {
		return Object.values( filter ).some( ( selected ) => selected );
	} );
```

Reply:
Thanks for the great suggestion. This makes the code more cleaner 👍